### ISSUE TITLE
Start the project from scratch.

### ISSUE THREAD

**Issue by h-maina** [abbra/cockpit-app-samba-ad#1]
Proposing that we start over from scratch due to:

- Outdated dependencies.

- React has made changes to some of the lifecycle methods and plans to depreciate them in future releases.

**Comment by abbra** [comment]
Yep. I replied to you on samba-technical@ mailing list, let's continue there.
https://lists.samba.org/archive/samba-technical/2020-March/135052.html
